ChaosMod/Effects/db/Misc/MiscStuffGuns.cpp

```diff
@@ -0,0 +1,154 @@
+#include <stdafx.h>
+
+static Vector3 DegToRadian(const Vector3& angles)
+{
+	return Vector3::Init(
+		angles.x * .0174532925199433F,
+		angles.y * .0174532925199433F,
+		angles.z * .0174532925199433F
+	);
+}
+
+static Vector3 GetCoordsFromGameplayCam(float distance)
+{
+	Vector3 rot = DegToRadian(GET_GAMEPLAY_CAM_ROT(2));
+	Vector3 coords = GET_GAMEPLAY_CAM_COORD();
+
+	rot.y = distance * cos(rot.x);
+	coords.x = coords.x + rot.y * std::sin(rot.z * -1.f);
+	coords.y = coords.y + rot.y * std::cos(rot.z * -1.f);
+	coords.z = coords.z + distance * sin(rot.x);
+
+	return coords;
+}
+
+static bool IsWeaponShotgun(Hash wepHash)
```

Comment:
Extract method to weapons.h

Reply:
do you mean WeaponPool.h? can't find a weapons.h

Reply:
my bad, i thought there already was a "Weapons.h"
I would create a new file ("Weapons.h") inside the "Util"-folder, move the function over there and include the header in stdafx.h